clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -545,8 +545,9 @@ getTripleBasedSYCLPostLinkOpts(const ArgList &Args,
   else
     PostLinkArgs.push_back("-spec-const=emulation");
 
-  if (Triple.isSPIROrSPIRV() || SYCLNativeCPU)
-    PostLinkArgs.push_back("-properties");
+  // TODO: If we ever pass -ir-output-only based on the triple,
+  // make sure we don't pass -properties.
+  PostLinkArgs.push_back("-properties");
```

Comment:
Shouldn't we update some driver tests?

Reply:
I do not think we have any test checking if '-properties' has been added in the new offload model. May be we can add one as part of this change?

Thanks

Reply:
This should already be locked down by tests such as [this](https://github.com/intel/llvm/blob/64a13200c43998043b6e240ab42bdc8e72982d26/clang/test/Driver/sycl-post-link-options.cpp#L20C33-L23C31) one, and we don't have any tests that checked we do not pass the flag in the previous code and now it's passed unconditionally, so I don't think there's a coverage missing or tests to add

Reply:
Merging now because non-spir targets with the new offload model were broken, happy to address any further feedback in followup PR